perf/src/sigverify.rs

```diff
@@ -336,7 +336,7 @@ fn check_for_simple_vote_transaction(
     // Packet should have at least 1 more byte for instructions.len
     let _ = instructions_len_offset
         .checked_add(1usize)
-        .filter(|v| *v <= packet.meta().size)
+        .filter(|v| *v <= packet.size())
```

Comment:
we're setting packet.meta().size right? why not keep using it for now? (here and
everywhere else))

Reply:
We set `meta.size` to `u64::MAX` in BytesPacket. I left a comment there why and we talked about it once privately.

https://github.com/vadorovsky/agave/blob/d076d981a9100770dbc79c3fa351620b595ede3e/perf/src/packet.rs#L52-L61

I see no point in manual tracking of the size, so I want to deprecate that field. And get rid of it once we can (not this PR).

Reply:
fair